### PR TITLE
feat(task-service): add mysql db migrations

### PR DESCRIPTION
**Pull request by sf-sahil-jassal**
add support for mysql db by creating seperate
directories for mysql and pg migrations

MIGRATION CHANGE:
migration-20231222070515- add seperate directory for mysql migrations

gh-1836

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #1836

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Intermediate change (work in progress)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist:

- [x] Performed a self-review of my own code
- [ ] npm test passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the style guide
- [ ] API Documentation in code was updated
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: services/task-service/migrations/mysql/migrations/sqls/20231222070515-init-up.sql [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+-- creating table tasks
+CREATE TABLE tasks (
+    id                  VARCHAR(36) NOT NULL PRIMARY KEY,
+    `key`               VARCHAR(255) NOT NULL,
+    name                VARCHAR(255),
+    description         VARCHAR(255),
+    status              VARCHAR(255),
+    priority            VARCHAR(255),
+    severity            VARCHAR(255),
+    type                VARCHAR(255),
```
Comment (yeshamavani) [inline services/task-service/migrations/mysql/migrations/sqls/20231222070515-init-up.sql:30]: can we change all of these to text

File: services/task-service/migrations/mysql/migrations/package.json
Comment (yeshamavani) [inline services/task-service/migrations/mysql/migrations/package.json:1]: not needed

File: services/task-service/migrations/mysql/migrations/sqls/package.json
Comment (yeshamavani) [inline services/task-service/migrations/mysql/migrations/sqls/package.json:1]: not needed

**Review by yeshamavani — CHANGES_REQUESTED** [review]

File: services/task-service/migrations/mysql/migrations/sqls/20231222070515-init-up.sql (hunk 1 above)
Comment (sf-sahil-jassal) [reply services/task-service/migrations/mysql/migrations/sqls/20231222070515-init-up.sql:30]: in psql migrations, only VARCHAR was written for these columns. So, I changed to that.

**Review by yeshamavani — APPROVED** [review]

**Pull request merged**